core/src/main/java/com/opentable/logging/CommonLogHolder.java

```diff
@@ -61,6 +61,23 @@
 
     private CommonLogHolder() { }
 
+    /**
+     * Mock out the environment.  You probably don't want to do this.
+     */
+    public static void setEnvironment(String otEnv, String otEnvType, String otEnvLocation, String otEnvFlavor) {
```

Comment:
Can we limit the publicity of this method?

Reply:
Not without requiring test cases that need this to go in the `logging` package.